gini/gini-vision-lib-ios
# Added analysis screen error view

When there is an error in the analysis or uploading a document, it has to be shown in the analysis screen. For the uploading doesn't apply for images when multipage is enabled, since the error will be shown in the multipage screen.
Also fixed minor issue with auto-layout in page cell.

### How to test
Run the Example app, disable the internet connection, scan a qr code and see that an error is shown in the analysis screen.

### Merging
Automatic

### Issue
#243
